src/pages/LoginPage.tsx

```diff
@@ -37,7 +37,7 @@ export const LoginPage: FC<LoginPageProps> = ({ flow }) => {
           setLoginFlow(flow);
         })
         .catch(e => {
-          console.log(e);
+          console.error(e);
```

Comment:
I thought we should not use the console as the feedback suggested? Will this error appear on production?